src/views/AddItem.css

```diff
@@ -0,0 +1,119 @@
+.timeframe {
+	display: flex;
+	flex-direction: column;
+}
+
+.timeframe > fieldset {
+	display: flex;
+	flex-direction: column;
+	padding: 20px;
+	margin: 50;
+	width: auto;
+	position: fixed;
+}
+
+.timeframe label {
+	padding: 10px;
+}
```

Comment:
Dropping a suggestion for the styling. Currently, the styling of the contents of an item is a bit off-center. You can use `display: flex` on the `.label` to get things to line up better. The spacing gets condensed a bit but you can use e.g. `gap: 10px` as well and they'll space out better.

Using `gap` on the `.timeframe` selector instead of 

```css
.timeframe label {
	padding: 10px;
}
```
also might work better.

https://user-images.githubusercontent.com/833231/187012137-e8761de2-9a2b-4127-9a99-26206a925374.mp4